docs/docs/guide/express-js.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: Express.js and GrowthBook
+description: This guide shows how to integrate GrowthBook with Express.js to manage server-side feature flags.
+slug: express-js
+---
+
+# Feature Flags with Express.js and GrowthBook
+
+This guide walks through how to add GrowthBook feature flags to an Express.js application. It assumes you are starting from scratch, so if you already have an Express.js application, you can skip to step 2.
+
+## 1. Create an Express.js app
+
+First, install Express.js
+
+```bash npm2yarn
+npm install --save express
+```
+
+Then, create an `index.js` file with a simple hello world route
+
+```js
+const express = require('express')
+const app = express()
+
+app.get('/', (req, res) => {
+  res.send('Hello World!')
+})
+
+// Listen on port 3000
+const port = 3000
+app.listen(port, () => {
+  console.log(`Example app listening on port ${port}`)
+})
+```
+
+Finally, run the app with:
+
+```bash
+node index.js
+```
+
+Visit `http://localhost:3000` and you should see a "Hello World!" response!
+
+## 2. Create a GrowthBook Account
+
+GrowthBook can be [self-hosted](/self-host), but for simplicity, we will use GrowthBook Cloud in this guide, which is free for simple projects like this.
+
+Go to [https://app.growthbook.io](https://app.growthbook.io) and create a new account if you don't have one yet.
+
+Once you are logged in, create an **SDK Connection** and select Node.js as the language. This will generate a unique **Client Key** for you. Keep note of this key as we will use it in the next step.
+
+## 3. Integrate the GrowthBook JavaScript SDK into the Express.js app
+
+Create the file `.env` if it doesn't exist yet and add the generated key there:
+
+```bash
+GROWTHBOOK_API_HOST=https://cdn.growthbook.io
+GROWTHBOOK_CLIENT_KEY=
+```
+
+Now lets install the GrowthBook JavaScript SDK
+
+```bash npm2yarn
+npm install --save @growthbook/growthbook
+```
+
+Lets now modify the top of `index.js` file and create a **GrowthBookClient** instance:
+
+```js
+const express = require('express')
```

Comment:
What's your thought about using commonjs vs esm here? Commonjs is what Express has in their docs, so I think it's fine here.

Reply:
yeah, I was copying the hello world from express